client/lib/media/library-selected-store.js

```diff
@@ -13,24 +13,25 @@ var MediaStore = require( './store' ),
 /**
  * Module variables
  */
-var _media = {},
```

Comment:
It gets out of the control when I export `Store` object as default, and add several `let` exports. It would work properly if we could import that stuff, but we use require and it returns object like:
```js
{ _media: {},
  _pointers: {},
  default: 
   { ... }
}
```
when you export:
```js
export let _media = {};
export let _pointer = {};
export default Store;
```